Makefile

```diff
@@ -6,14 +6,15 @@ PKG := $(ORG)/rukpak
 GO_INSTALL_OPTS ?= "-mod=readonly"
 export IMAGE_REPO ?= quay.io/operator-framework/rukpak
 export IMAGE_TAG ?= latest
+export GO_BUILD_TAGS ?= downstream
```

Comment:
We should _really_ try to avoid maintaining upstream/downstream differences in shared files. I have a feeling that will become unmanageable and result in:
- merge conflicts
- issues related to manually resolving merge conflicts
- losing track of important upstream changes because we've taken downstream in a different direction
- a slow creep of accepting _more_ of these sorts of downstream patches to upstream files, potentially even in Go code, thus exacerbating the first three problems.

Instead, I imagined that we'd just call our upstream Makefile like `GO_BUILD_TAGS=downstream make` to override this value rather than changing its default.

Reply:
If we separate these things out, it will also be _much_ easier to understand what changes came from upstream vs. what changes are downstream-only.